### ISSUE TITLE
Submit not working with reCAPTCHA v2 Invisible

### ISSUE THREAD

**Issue by karthikkstar** [verbb/formie#916]
**Description**
Initial formie worked with reCAPTCHA v2 Invisible. We noticed now formie not working with reCAPTCHA v2 Invisible.

**Note**: If we disable the reCAPTCHA integration, Form submission perfectly working. Also its working with reCAPTCHA v3. 
only problem with reCAPTCHA v2.

**Steps to reproduce**
1. Click the submit button without filing any input field, Then the validation error will appear.
2. Fill all the input fields and submit the form. Submit button disabled and loader loading. Nothing happened on the browser network tab.  

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax 
- Client-side Validation: Yes
- Custom Form Templates:  No

**Additional info**
- Plugin version:1.5.18
- Craft version: 3.7.43
- Multi-site: Yes

**Comment by engram-design** [comment]
Is this the invisivle captcha? There should be no validation error trigger for the invisible captcha, only if you have required fields blank (which isn't captcha related), or if you use the V2 checkbox, and left un-checked.

Can you confirm? I've just tested the V2 invisible captcha, and it's working as it should. Is there any JS error when you load the form, or when submitting? 

Is this a multi-page form? You didn't state if it was.

**Comment by engram-design** [comment]
Ah, I see - if you have required fields, then submit, then try to submit again, the submit button sits in a disabled state. I'll investigate.

**Comment by karthikkstar** [comment]
@engram-design Yes. We using Invisible reCAPTCHA V2. 
Yes. We have required fields. 
No JS error on load and submitting

Is this a multi-page form? - NO.

Thank you.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by karthikkstar** [comment]
Thank you @engram-design

**Comment by engram-design** [comment]
Fixed in [1.5.19](https://github.com/verbb/formie/releases/tag/1.5.19)
